client/src/component/WrappedTicket/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { Link, Route } from 'react-router-dom'
+import './style.css'
+class WrappedTicket extends Component {
+    state = {
+        wrappedTickets: [],
+    }
+    componentDidMount() {
+        this.setState({
+            wrappedTickets: [{
+                id: 1,
+                email: 'jamalat@gmail.com',
+                description: 'This is a sample ticket with some sample description',
+                date: 'April 9'
+                , subject: "subject"
+            },
+            {
+                id: 2,
+                email: 'jamalat@gmail.com',
+                description: 'This is a sample ticket with some sample description',
+                date: 'April 9'
+                , subject: "subject"
+            }, {
+                id: 3,
+                email: 'jamalat@gmail.com',
+                description: 'This is a sample ticket with some sample description',
+                date: 'April 9'
+                , subject: "subject"
+            }, {
+                id: 4,
+                email: 'jamalat@gmail.com',
+                description: 'This is a sample ticket with some sample description',
+                date: 'April 9'
+                , subject: "subject"
+            }]
+        })
+    }
+    render() {
+        if (this.state.wrappedTickets) {
```

Comment:
There's a cleaner way to do this render.
https://reactjs.org/docs/conditional-rendering.html

Reply:
This is just for your information. It would make the code cleaner, and you will not have to use ```return " "```, If you don't want to do it, then don't. It's not a big deal.